chapter10/efs-backup.yaml

```diff
@@ -233,10 +238,7 @@ Resources:
 
           # install dependencies
           yum install -y nc amazon-efs-utils
```

Comment:
`nc` not needed anymore?

Reply:
Nope, the `efs-utils` tool uses the API to resolve the hostname. And the CloudFormation `DependsOn` makes sure the mount target has been created before the instance starts.